modules/transport-netty4/src/main/java/org/elasticsearch/http/netty4/Netty4HttpServerTransport.java

```diff
@@ -289,7 +289,7 @@ protected void stopInternal() {
     }
 
     @Override
-    protected void onException(HttpChannel channel, Exception cause) {
+    public void onException(HttpChannel channel, Exception cause) {
         if (cause instanceof ReadTimeoutException) {
```

Comment:
for uniformity, should we convert Netty's ReadTimeoutException here to our own HttpReadTimeoutException and then leave the exception handling to the super call?